components/kyma-environment-broker/internal/process/input/input.go

```diff
@@ -268,6 +278,15 @@ func (r *RuntimeInput) applyGlobalOverridesForUpgradeRuntime() error {
 	return nil
 }
 
+func (r *RuntimeInput) addRandomStringToTrialRuntimeName() error {
+	rand.Seed(time.Now().UnixNano())
+	if r.provisioningParameters.PlanID == broker.TrialPlanID {
```

Comment:
there is an IsTrial func in the broker package

Reply:
Why the random part is used only for trial? are there any business reasons?

Reply:
User cannot specify the name when provisioning a trial skr. A random name comes from CP and there were occurences when the name was repeated. We want to make sure that the name is random and it won't cause any problems in the Director DB